ci/test_cpp.sh

```diff
@@ -32,6 +31,8 @@ rapids-mamba-retry install \
 rapids-logger "Check GPU usage"
 nvidia-smi
 
+EXITCODE=0
+trap "EXITCODE=1" ERR
```

Comment:
I can't leave a suggestion on the particular line in GitHub, but it looks like there's a `TODO` a few lines down that can be removed here.

After that is removed, the whitespace can also be removed between the logger statement and subsequent `for` loop.